modules/manage/partials/iceberg/about-iceberg-topics.adoc

```diff
@@ -327,6 +327,8 @@ The DLQ table itself uses the `key_value` schema, consisting of two columns: the
 
 You can inspect the DLQ table for records that failed to write to the Iceberg table, and you can take further action on these records, such as transforming and reprocessing them, or debugging issues that occurred upstream.
 
+NOTE: Topic property misconfiguration, such as setting `redpanda.iceberg.mode` to `value_schema_latest` but not specifying the fully qualified schema name, does not cause records to be written to the DLQ table. Instead, Redpanda pauses the topic data translation to the Iceberg table until you fix the misconfiguration.
```

Comment:
you don't need to specify the schema name if your schema uses the TopicNamingStrategy thoug: https://docs.redpanda.com/current/manage/iceberg/choose-iceberg-mode/#override-value-schema-latest-default

IIUC you have a choice of doing that or specifying the fully qualified name, so this is misleading.

At least the docs say that.  cc @rockwotj